#### Assignment stats - 3

##### 1. Blood glucose levels for obese patients have a mean of 100 with a standard deviation of 15. A researcher thinks that a diet high in raw cornstarch will have a positive effect on blood glucose levels. A sample of 36 patients who have tried the raw cornstarch diet have a mean glucose level of 108. Test the hypothesis that the raw cornstarch had an effect or not.

In [2]:
import numpy as np

In [3]:

# H0: mew= 100
# H1: mew > 100

mew = 100
std_dev = 15
n=36
x_bar = 108
alpha = 0.05

z = (x_bar - mew)/std_dev*np.sqrt(n)
z

3.2

You can look at the probability by looking at z- table and p-value associated with 3.20 is 0.9993 i.e. probability of having value less than 108 is 0.9993 and more than or equals to 108 is (1-0.9993)=0.0007.

It is less than 0.05 so we will reject the Null hypothesis i.e. there is raw cornstarch effect.

##### 2. In one state, 52% of the voters are Republicans, and 48% are Democrats. In a second state, 47% of the voters are Republicans, and 53% are Democrats. Suppose a simple random sample of 100 voters are surveyed from each state. What is the probability that the survey will show a greater percentage of Republican voters in the second state than in the first state?

For this analysis, 

let P1 = the proportion of Republican voters in the first state, 

P2 = the proportion of Republican voters in the second state, 

p1 = the proportion of Republican voters in the sample from the first state, and 

p2 = the proportion of Republican voters in the sample from the second state. 

The number of voters sampled from the first state (n1) = 100, and the number of voters sampled from the second state (n2) = 100.

The solution involves four steps.

Make sure the sample size is big enough to model differences with a normal population. Because 

n1P1 = 100 * 0.52 = 52, n1(1 - P1) = 100 * 0.48 = 48, n2P2 = 100 * 0.47 = 47

n2(1 - P2) = 100 * 0.53 = 53 are each greater than 10, the sample size is large enough.

Find the mean of the difference in sample proportions: E(p1 - p2) = P1 - P2 = 0.52 - 0.47 = 0.05.

Find the standard deviation of the difference.
σd = sqrt{ [ P1(1 - P1) / n1 ] + [ P2(1 - P2) / n2 ] }
σd = sqrt{ [ (0.52)(0.48) / 100 ] + [ (0.47)(0.53) / 100 ] }
σd = sqrt (0.002496 + 0.002491) = sqrt(0.004987) = 0.0706


Find the probability. This problem requires us to find the probability that p1 is less than p2. This is equivalent to finding the probability that p1 - p2 is less than zero. To find this probability, we need to transform the random variable (p1 - p2) into a z-score. That transformation appears below.


z p1 - p2 = (x - μ p1 - p2 ) / σd = = (0 - 0.05)/0.0706 = -0.7082
Using Stat Trek's Normal Distribution Calculator, we find that the probability of a z-score being -0.7082 or less is 0.24.


Therefore, the probability that the survey will show a greater percentage of Republican voters in the second state than in the first state is 0.24.

##### 3. You take the SAT and score 1100. The mean score for the SAT is 1026 and the standard deviation is 209. How well did you score on the test compared to the average test taker?

In [5]:
xi = 1100
x_mean = 1026
std_dev = 209

z = (xi-x_mean)/std_dev
print(z)

# This means that your score was .354 std devs above the mean

0.35406698564593303
